### DISCUSSION TITLE
Next 13 App route

### DISCUSSION THREAD

**Discussion by HighError** [avitorio/outstatic#122] [Q&A] [answered]
Is this CMS compatible with Next js 13 App route? And will there be instructions for launching it, if so.

**Comment by avitorio** [comment]
Yes. Your site can use the app directory, but the outstatic install should be in `/pages/`. We will be launching aversion where outstatic lives inside of the app directory in the near future.

**Reply by avitorio** [reply, marked as answer]
Hey @HighError, just to let you know. Our current 1.0-canary.1 release works fully inside the /app directory.
